assets/out

```diff
@@ -36,7 +36,7 @@ echo "Preparing SonarQube scanner..."
 
 scanner_opts=""
 
-project_path=$(jq -r '.params.project_path // ""' < "${payload}")
+project_path=/tmp/build/put/$(jq -r '.params.project_path // ""' < "${payload}")
```

Comment:
I was of the impression, that `$1` (The first argument passed to the script, which is assigned to the variable `$source`) would be a better solution than a hard-coded `/tmp/build/put/`.

Did I mis-interpret the documentation or am I missing something here?

Reply:
Probably yes, I'm going to test it ;)